Models/models.js

```diff
@@ -30,3 +31,24 @@ exports.selectArticleById = (article_id) => {
     }
   });
 };
+exports.selectCommentsByArticle = (article_id, sort_by = "created_at") => {
+  const SQL = `SELECT comment_id, votes, created_at, author, body FROM comments
+  WHERE  article_id = $1
+  ORDER BY ${sort_by} DESC;`;
+  return db.query(SQL, [article_id]).then((result) => {
+    return result.rows;
+  });
+};
+exports.checkIfArticleExists = (article_id) => {
+  const SQL = `SELECT * FROM articles WHERE article_id =$1;`;
+  return db.query(SQL, [article_id]).then((result) => {
+    if (result.rowCount === 0) {
```

Comment:
This is spot on - however you don't actually need a custom model function for this - you can just use your `selectArticleById` that you made for ticket 5 as it does exactly the same thing